03-postgres-typeorm/.env

```diff
@@ -0,0 +1,8 @@
+MONGODB_URI=
```

Comment:
It would be great if you can add the env file gitignore and change it to sample.env